include/grl/kuka/KukaDriver.hpp

```diff
@@ -261,6 +261,49 @@ namespace grl { namespace robot { namespace arm {
         }
    }
 
+  bool setPositionControlMode()
```

Comment:
Sure, its better that way. Due to time constraints I just added new named methods.